scripts/setup-env.js

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2018, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+/* eslint-disable consistent-return, no-console */
+
+'use strict';
+
+const fs = require('fs');
+const { execSync } = require('child_process');
+const path = require('path');
+
+function updateConfig(directory) {
+  if (!process.env.ISSUER || !process.env.CLIENT_ID || !process.env.CLIENT_SECRET || !process.env.USERNAME || !process.env.PASSWORD) {
+    console.log('[ERROR] Please set the necessary Environment variables (ISSUER, CLIENT_ID, CLIENT_SECRET, USERNAME, PASSWORD)');
+    process.exit(1);
+  }
+
+  const envFile = path.join(directory, '.env');
+  const data = fs.readFileSync(envFile, 'utf8');
+
+  let clientId = process.env.CLIENT_ID;
+  // For resource server, we need to set client_id of SPA in .env
+  if (directory.includes('resource-server')) {
+    if (!process.env.SPA_CLIENT_ID) {
+      console.error('[ERROR] Please set the SPA_CLIENT_ID for resource-server tests');
```

Comment:
Again, why can we not use the same application?  This worked fine for my testing?

Reply:
Try to copy `.env` that you used for `custom-login` into `resource-server`. I don't think it'll work